README.md

```diff
@@ -34,6 +34,9 @@ sensor/
 ...
 ```
 
+###
```

Comment:
Which is the title of this?

Reply:
Removed in https://github.com/erlerobot/HRIM/commit/97961b643b01446a190fedfa1460030c3743c46e#diff-04c6e90faac2675aa89e2176d2eec7d8L37